src/Views/EthernetPage.vala

```diff
@@ -30,14 +30,20 @@ namespace Network.Widgets {
         }
 
         construct {
+            widgets_stack = new Gtk.Stack ();
+            widgets_stack.visible = true;
```

Comment:
Since `show_all` is called elsewhere, it's not necessary to explicitly mark this as visible.

Also, it would probably be more organized to declare `widgets_stack` down on line 44 when we start to use it